### PR TITLE
xenvm_gicv3: Enable config parameters in gicv3 to match xenvm board

### PR DESCRIPTION
**Pull request by oleksiimoisieiev**
The same config paramters should be enabled in xenvm_gicv3 as in xenvm. Otherwise there is no posibility to build it.

### CODE REVIEW

File: boards/arm64/xenvm/xenvm_gicv3_defconfig [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
 
+# Enable PV hypervisor console
+CONFIG_UART_XEN_HVC_CONSOLEIO=y
+
 # Enable logging subsys
 CONFIG_LOG=y
 CONFIG_LOG_MODE_MINIMAL=n
 
-CONFIG_USERSPACE=n
+CONFIG_TEST=y
```
Comment (xakep-amatop) [inline boards/arm64/xenvm/xenvm_gicv3_defconfig:25]: (question) could you please move all below config to application config file?
do we really need them here?

Comment (oleksiimoisieiev) [reply boards/arm64/xenvm/xenvm_gicv3_defconfig:25]: Sure I can. But I've just copied it from xenvm_defconfig. If moving something to app - then all configs should be updated. Why did those parameters where added to xenvm and xenvm-qemu in the first place?

Comment (xakep-amatop) [reply boards/arm64/xenvm/xenvm_gicv3_defconfig:25]: I think it was a mistake and probably because people want to have fast crutch solutions

Comment (oleksiimoisieiev) [reply boards/arm64/xenvm/xenvm_gicv3_defconfig:25]: Why It was a mistake? I don't understand

Comment (xakep-amatop) [reply boards/arm64/xenvm/xenvm_gicv3_defconfig:25]: somebody don't want to use this config or for example shell config, why should board have it for all samples/applications?
I believe that board's configs may contain only minimal configuration 
board's configs should be universal and other stuff should be handled by configs of apps/samples/tests

**Comment by oleksiimoisieiev** [comment]
Closing as not relevant anymore

**Pull request closed without merging**